tests/std/tests/P0881R7_stacktrace/test.cpp

```diff
@@ -20,6 +23,34 @@
 
 using namespace std;
 
+#if defined(__cpp_lib_concepts) // TRANSITION, GH-395
+template <class Ostream, class Alloc = allocator<stacktrace_entry>>

```

Comment:
No change requested: I believe this should technically include `<memory>` for `allocator` (surprisingly, `<memory_resource>` isn't guaranteed to include `<memory>`), but given that in practice `<string>` etc. will always drag it in, and everything else looks great, this isn't worth resetting testing.